src/cformat.jl

```diff
@@ -1,16 +1,80 @@
 formatters = Dict{ Compat.ASCIIString, Function }()
 
-function sprintf1( fmt::Compat.ASCIIString, x )
+@static if VERSION >= v"0.6.0-dev.1671"
```

Comment:
OK, that's good to know (it might be nice to have the docstring for `@static` mention that)
In my own code, I'd just replaced any of the old macros (`@windows` etc. with `@static if ...` everwhere, and didn't realize that the `@static` part was not needed at the top level (does it hurt anything though to leave it in?)